collectors/python.d.plugin/smartd_log/smartd_log.chart.py

```diff
@@ -330,7 +331,13 @@
         'lines': [],
         'attrs': [ATTR233],
         'algo': ABSOLUTE,
-    }
+    },
+    'nand_writes_1gib': {
+        'options': [None, 'NAND Writes 1GiB', 'blocks', 'wear', 'smartd_log.nand_writes_1gib', 'line'],
+        'lines': [],
+        'attrs': [ATTR249],
+        'algo': ABSOLUTE,
```

Comment:
from https://en.wikipedia.org/wiki/S.M.A.R.T.

> Total NAND Writes. Raw value reports the number of writes to NAND in 1 GB increments.[77]


Perhaps i am missing something, can't say i understand this attribute 😄 


- why units is `blocks`? 
- does absolute work? we are interested in total number of those _writes_ and not _writes_ rate (incremental)?

Reply:
Hm, you're right! I was copying blindly what was up and was not sure if it was right.